downloader/src/main/kotlin/vcs/Mercurial.kt

```diff
@@ -123,12 +123,9 @@ class Mercurial : VersionControlSystem(), CommandLineTool {
             log.info { "Configuring Mercurial to do sparse checkout of path '${vcs.path}'." }
 
             // Mercurial does not accept absolute paths.
-            val licenseFileGlobs = ALL_LICENSE_FILENAMES.map { "**/$it" }
+            val globPatterns = ALL_LICENSE_FILENAMES.map { "**/$it" } + "${vcs.path}/**"
```

Comment:
Nit: Can we make `"${vcs.path}/**"` go first here to resemble the original code more closely, and to align with the order in the `Git` class to make the VCS path go frist, and then list the license file patterns?

Reply:
I realize that's not possible due to the `+` operator to operate on the list.